src/components/icons/TwitterIcon.tsx

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
```

Comment:
nit, could we take the twitter icon from somewhere like https://feathericons.com/ ? so we dont have to manually type the path